Sample Applications/DataBindingDemo/App.xaml

```diff
@@ -12,9 +12,11 @@
 
             <local:DateConverter x:Key="DateConverter" />
 
-            <DataTemplate DataType="{x:Type local:AuctionItem}" >
+            <DataTemplate DataType="{x:Type local:AuctionItem}">
                 <Border Style="{StaticResource AuctionItemBorderStyle}" Name="border">
-                    <Grid>
+                    <Grid Style="{StaticResource GridStyle}"
+                           Name="AuctionItemsGrid"
+                           MouseDown="OnGridFocus">
```

Comment:
You should re-think this.  You can style the TextBoxes to bind triggers to their ancestor control's MouseDown by type (say Grid) and then you can react to changes in the parent.  Also, it seems you've named the code OnGridFocus, but are using MouseDown.  Which did you want?

Reply:
Incroporated trigger on ListBoxItem instead of codebehind